src/pretix/control/templates/pretixcontrol/item/include_addons.html

```diff
@@ -8,8 +8,7 @@
         workshops as add-ons to the conference ticket. With this configuration, the workshops cannot be bought
         on their own but only in combination with a conference ticket. You can here specify categories of products
         that can be used as add-ons to this product. You can also specify the minimum and maximum number of
-        add-ons of the given category that can or need to be chosen. The user can buy every add-on from the
-        category at most once. If an add-on product has multiple variations, only one of them can be bought.
+        add-ons of the given category that can or need to be chosen.
     {% endblocktrans %}
```

Comment:
For a few days this limitation has been removed. Feel free to update this explanation further.

Reply:
:+1: